homeassistant/components/device_tracker/bmw_connected_drive.py

```diff
@@ -0,0 +1,52 @@
+"""Device tracker for BMW Connected Drive vehicles.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/device_tracker.bmw_connected_drive/
+"""
+import logging
+
+from homeassistant.components.bmw_connected_drive import DOMAIN \
+    as BMW_DOMAIN
+from homeassistant.util import slugify
+
+DEPENDENCIES = ['bmw_connected_drive']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_scanner(hass, config, see, discovery_info=None):
+    """Set up the BMW tracker."""
+    accounts = hass.data[BMW_DOMAIN]
+    _LOGGER.debug('Found BMW accounts: %s',
+                  ', '.join([a.name for a in accounts]))
+    for account in accounts:
+        for vehicle in account.account.vehicles:
+            tracker = BMWDeviceTracker(see, account, vehicle)
+            account.add_update_listener(tracker.update)
+            tracker.update()
+    return True
+
+
+class BMWDeviceTracker(object):
+    """BMW Connected Drive device tracker."""
+
+    def __init__(self, see, account, vehicle):
+        """Initialize the Tracker."""
+        self._see = see
+        self.vehicle = vehicle
+        self._account = account
```

Comment:
This doesn't seem to be used.

Reply:
Ah yes, you have a good eye for these things...